pytools/prefork.py

```diff
@@ -164,40 +210,55 @@ def _remote_invoke(self, name, *args, **kwargs):
         assert status == "ok"
         return result
 
-    def _quit(self):
+    def _quit(self) -> None:
         self._remote_invoke("quit")
 
         from os import waitpid
         waitpid(self.server_pid, 0)
 
-    def call(self, cmdline, cwd=None):
-        return self._remote_invoke("call", cmdline, cwd)
+    def call(self, cmdline: Sequence[str], cwd: str | None = None) -> int:
+        result = self._remote_invoke("call", cmdline, cwd)
+
+        assert isinstance(result, int)
+        return result
 
-    def call_async(self, cmdline, cwd=None):
-        return self._remote_invoke("call_async", cmdline, cwd)
+    def call_async(self, cmdline: Sequence[str], cwd: str | None = None) -> int:
+        result = self._remote_invoke("call_async", cmdline, cwd)
+
+        assert isinstance(result, int)
+        return result
 
-    def call_capture_output(self, cmdline, cwd=None, error_on_nonzero=True):
-        return self._remote_invoke("call_capture_output", cmdline, cwd,
+    def call_capture_output(self,
+                            cmdline: Sequence[str],
+                            cwd: str | None = None,
+                            error_on_nonzero: bool = True,
+                            ) -> tuple[int, bytes, bytes]:
+        return self._remote_invoke("call_capture_output", cmdline, cwd,  # type: ignore[no-any-return,unused-ignore]
```

Comment:
This is a bit annoying.. the non-strict mypy complains about the `unused-ignore` and the strict mypy complains about the `no-any-return` :(

Reply:
You could change the `Any`s to `object`s maybe?